nikola/plugins/template/jinja.py

```diff
@@ -52,7 +52,10 @@ def __init__(self):
         """Initialize Jinja2 environment with extended set of filters."""
         if jinja2 is None:
             return
-        self.lookup = jinja2.Environment()
+        # Not ideal because it's not using our cache structure
```

Comment:
According to the jinja docs "system temp folder". It looks like it needs to be set on env creation. Maybe we can delay that until set_site?